### DISCUSSION TITLE
Where are the pin definitions for the audio examples?

### DISCUSSION THREAD

**Discussion by richardklingler** [hathach/tinyusb#1302] [Q&A]
Good morning

Installed tinyusb and pico.sdk yesterday and compiled some examples...for example the uac2 headphone as I want to use the RP2040 for an usb speaker project....

But now my question:

Where are the pin definitions for the audio inputs/outputs like BCLK, WCLK, MCLK, DATA and so on?
Don't see anything at all in the sources....

Or do the examples don't use any real in/outputs?

thanks in advance
richard

**Comment by hathach** [comment]
> Or do the examples don't use any real in/outputs?

No, these examples only focus on the USB side and its proof of concept. Anything that is not USB such as I2S, DAC are not implemented.
